src/index.html

```diff
@@ -6,13 +6,76 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./styles/card.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/stars.scss"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="../src/images/imac.jpeg"
+        alt="imac"
+        class="card__product-image"
+      />
+
+      <h3 class="card__product-name">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <h4 class="card__product-code">Product code: 195434</h4>
+
+      <div class="card__product-rating">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <h4 class="card__product-rating-review">Reviews: 5</h4>
+      </div>
+
+      <div class="card__product-price">
+        <h3 class="card__product-price-text">Price:</h3>
```

Comment:
The word 'Price:' is not a heading, it's just a piece of data. It's better to use a `p` or `span` tag here. Headings are used to structure the content in a meaningful way. They represent the document outline.